site/docs/toolchains.md

```diff
@@ -361,33 +361,39 @@ This will end up building `//bar_tools:barc_linux` but not
 **Note:** Some Bazel rules do not yet support toolchain resolution.
 
 For each target that uses toolchains, Bazel's toolchain resolution procedure
-determines the target's concrete dependencies. The procedure takes as input a
-set of required toolchain types, the target platform, a list of available
-execution platforms, and a list of available toolchains. Its outputs are a
+determines the target's concrete toolchain dependencies. The procedure takes as input a
+set of required toolchain types, the target platform, the list of available
+execution platforms, and the list of available toolchains. Its outputs are a
 selected toolchain for each toolchain type as well as a selected execution
 platform for the current target.
 
 The available execution platforms and toolchains are gathered from the
 `WORKSPACE` file via
-[`register_execution_platforms`](https://docs.bazel.build/versions/master/skylark/lib/globals.html#register_execution_platforms)
+[`register_execution_platforms`](skylark/lib/globals.html#register_execution_platforms)
 and
-[`register_toolchains`](https://docs.bazel.build/versions/master/skylark/lib/globals.html#register_toolchains).
+[`register_toolchains`](skylark/lib/globals.html#register_toolchains).
 Additional execution platforms and toolchains may also be specified on the
 command line via
-[`--extra_execution_platforms`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--extra_execution_platforms)
+[`--extra_execution_platforms`](command-line-reference.html#flag--extra_execution_platforms)
 and
-[`--extra_toolchains`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--extra_toolchains).
+[`--extra_toolchains`](command-line-reference.html#flag--extra_toolchains).
 The host platform is automatically included as an available execution platform.
 Available platforms and toolchains are tracked as ordered lists for determinism,
-with preference given to earlier items in the list.
 
 The resolution steps are as follows.
 
-1. For each available execution platform, we associate each toolchain type with
+1. If the target being configured specifies the
```

Comment:
I'd remove "being configured", it's clear from context what target we're talking about. Even rule authors don't always know exactly what's entailed by the configuration phase or think in terms of "configured targets".

Reply:
Done.